src/validations/personName.ts

```diff
@@ -0,0 +1,44 @@
+import { body } from "express-validator";
+
+export const nameValidator = [
+    body("first-name").custom((value, { req }) => firstNameChecker(req.body["first-name"], req.body["last-name"])).bail().isLength({ max: 50 }).withMessage("invalidFirstNameLength"),
```

Comment:
I saw that you have declared a constant for max length. May be that can be used here instead of 50